benchmarks/src/main/java/io/grpc/benchmarks/driver/LoadClient.java

```diff
@@ -0,0 +1,495 @@
+              true,
+              Utils.DEFAULT_FLOW_CONTROL_WINDOW,
+              false);
+    }
+
+    // Create a stub per channel
+    if (config.getClientType() == Control.ClientType.ASYNC_CLIENT) {
+      asyncStubs = new BenchmarkServiceGrpc.BenchmarkServiceStub[channels.length];
+      for (int i = 0; i < channels.length; i++) {
+        asyncStubs[i] = BenchmarkServiceGrpc.newStub(channels[i]);
+      }
+    } else {
+      blockingStubs = new BenchmarkServiceGrpc.BenchmarkServiceBlockingStub[channels.length];
+      for (int i = 0; i < channels.length; i++) {
+        blockingStubs[i] = BenchmarkServiceGrpc.newBlockingStub(channels[i]);
+      }
+    }
+
+    // Determine no of threads
+    threadCount = config.getClientType() == Control.ClientType.SYNC_CLIENT
+        ? config.getOutstandingRpcsPerChannel() * config.getClientChannels() :
+        config.getAsyncClientThreads();
+    // Use a fixed sized pool of daemon threads.
+    fixedThreadPool = Executors.newFixedThreadPool(threadCount,
+        new DefaultThreadFactory("client-worker", true));
+
+    // Create the load distribution
+    if (config.getLoadParams().getClosedLoop() != null) {
+      distribution = new EnumeratedIntegerDistribution(new int[]{0}, new double[]{1});
+    } else if (config.getLoadParams().getPoisson() != null) {
+      new PoissonDistribution(config.getLoadParams().getPoisson().getOfferedLoad() / threadCount);
+    } else  {
+      throw new UnsupportedOperationException();
+    }
+
+    // Create payloads
+    switch (config.getPayloadConfig().getPayloadCase()) {
+      case SIMPLE_PARAMS: {
+        Payloads.SimpleProtoParams simpleParams = config.getPayloadConfig().getSimpleParams();
+        simpleRequest = Utils.makeRequest(Messages.PayloadType.COMPRESSABLE,
+            simpleParams.getReqSize(), simpleParams.getRespSize());
+        break;
+      }
+      case BYTEBUF_PARAMS: {
+        PooledByteBufAllocator alloc = PooledByteBufAllocator.DEFAULT;
+        genericRequest = alloc.buffer(config.getPayloadConfig().getBytebufParams().getRespSize());
+        if (genericRequest.capacity() > 0) {
+          genericRequest.writerIndex(genericRequest.capacity() - 1);
+        }
+        break;
+      }
+      default: {
+        // Not implemented yet
+        throw new IllegalArgumentException("Scenario not implemented");
+      }
+    }
+
+    // Create the histogram recorder
+    recorder = new Recorder((long) config.getHistogramParams().getMaxPossible(), 3);
+  }
+
+  /**
+   * Start the load scenario.
+   */
+  void start() {
+    Runnable r;
+    for (int i = 0; i < threadCount; i++) {
+      r = null;
+      switch (config.getPayloadConfig().getPayloadCase()) {
+        case SIMPLE_PARAMS: {
+          if (config.getClientType() == Control.ClientType.SYNC_CLIENT) {
+            if (config.getRpcType() == Control.RpcType.UNARY) {
+              r = new BlockingUnaryWorker(blockingStubs[i % blockingStubs.length]);
+            }
```

Comment:
We dont generate methods in blocking stubs for streaming calls, added no real API value as you always had to have callbacks